runtime/compiler/env/j9method.cpp

```diff
@@ -1985,13 +1985,14 @@ TR_J9Method::TR_J9Method()
    {
    }
 #endif /* defined(J9VM_OPT_JITSERVER) */
-
```

Comment:
Please add back this blank line that was accidentally removed.  It makes it a little easier to read.